src/uniprotkb/components/protein-data-views/EMBLView.tsx

```diff
@@ -11,7 +11,22 @@ import * as logging from '../../../shared/utils/logging';
 import { Xref } from '../../../shared/types/apiModel';
 import { PropertyKey } from '../../types/modelTypes';
 
-import EMBLXrefProperties from '../../config/emblXrefPropertiesData.json';
+const EMBLXrefProperties: Record<string, string> = {
```

Comment:
Everything looks fine. I couldn't connect this one. Is this related to lineage? (out of my ignorance)

Reply:
Ah, sorry no, not related. It was to remove a small imported file because that way it also gave more information to TS about its content (see removed `as` below).
But that got into this change and I forgot to put it somewhere else.